app/assets/javascripts/discourse-plugins/index.js

```diff
@@ -243,11 +247,6 @@ module.exports = {
     return true;
   },
 
-  treeFor() {
-    // This addon doesn't contribute any 'real' trees to the app
-    return;
-  },
-
```

Comment:
Now that it has a child addon, the discourse-plugins addon does have a tree to return to the app, so we need to remove this override.